tools/os_deps/deps.yum.RHEL8

```diff
@@ -21,4 +23,4 @@ sqlite-devel
 which
 Xvfb
 zlib
-zlib-devel
+izlib-devel
```

Comment:
@santos1709 what's the difference between this and the old `zllib-devel`

Reply:
@santos1709  the PR looks good, but since it changes the lib requirements will need to consult with others to make sure they are ok with adding these cc: @jhou1, @liangxia do you have objects to these library addition?

Reply:
@pruan-rht ohhh this is actually a typo